importer/importer_utils.py

```diff
@@ -306,7 +306,15 @@ def get_text_inside_brackets(text):
     if "(" in text:
         first_bracket_index = text.find('(')
         last_bracket_index = text.rfind(')')
-        return text[first_bracket_index + 1:last_bracket_index]
+        result = text[first_bracket_index + 1:last_bracket_index]
+        if "(" in result or ")" in result:
+            for character in result:
```

Comment:
"(" is okay because it's okay to have some brackets in the result, if we allow for nested brackets. But if the first bracket in the result is a closing one -- `foo)` -- then something went wrong.